lastuser_core/utils.py

```diff
@@ -13,29 +13,29 @@
 
 
 def make_redirect_url(url, use_fragment=False, **params):
-    urlparts = list(urlparse.urlsplit(url))
+    urlparts = list(urllib.parse.urlsplit(url))
     # URL parts:
     # 0: scheme
     # 1: netloc
     # 2: path
     # 3: query -- appended to
     # 4: fragment
-    queryparts = urlparse.parse_qsl(
+    queryparts = urllib.parse.parse_qsl(
         urlparts[4] if use_fragment else urlparts[3], keep_blank_values=True
     )
     queryparts.extend([(k, v) for k, v in params.items() if v is not None])
     queryparts = [
         (
-            key.encode('utf-8') if isinstance(key, unicode) else key,
-            value.encode('utf-8') if isinstance(value, unicode) else value,
+            key.encode('utf-8') if isinstance(key, str) else key,
+            value.encode('utf-8') if isinstance(value, str) else value,
```

Comment:
Is `.encode` still required with `urllib.parse`?

Reply:
Seems so. works if we send string as well, but then urllib encodes it anyway.
<img width="424" alt="Screen Shot 2020-02-20 at 10 47 33 AM" src="https://user-images.githubusercontent.com/357253/74903253-6d479e80-53ce-11ea-99f3-5f494c53e2bc.png">

Reply:
This is the only pending issue. I don't know enough to comment, so your call. If this is working, this PR can be merged.

Reply:
This function has a test and that's passing. https://github.com/hasgeek/lastuser/blob/84f74635a983dfb101944e2c5f322ea8198878e0/tests/unit/lastuser_core/test_utils.py#L27